LICENSE.sig

```diff
@@ -0,0 +1,95 @@
+-----BEGIN PGP MESSAGE-----
```

Comment:
What is this file?

Reply:
this was definitely not meant to be committed, I was testing the agents ability to sign something. This has been removed.